src/mlpack/core/math/multiply_slices.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file core/math/multiply_slices.hpp
+ * @author Mrityunjay Tripathi
+ *
+ * Function to perform matrix multiplication on cubes.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_MATH_MULTIPLY_SLICES_HPP
+#define MLPACK_CORE_MATH_MULTIPLY_SLICES_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace math /** Miscellaneous math routines. */ {
+
+template <typename CubeType>
+CubeType MultiplyCube2Cube(const CubeType& cubeA,
+                           const CubeType& cubeB,
+                           const bool ATranspose = false,
+                           const bool BTranspose = false);
+
+template <typename MatType, typename CubeType>
+CubeType MultiplyMat2Cube(const MatType& matA,
+                          const CubeType& cubeB,
+                          const bool ATranspose = false,
+                          const bool BTranspose = false);
```

Comment:
According to the style guide arguments should begin with a lowercase letter.
```suggestion
                          const bool aTranspose = false,
                          const bool bTranspose = false);
```